.github/bot_templates/MARKER_REPORT_ISSUE.md.j2

```diff
@@ -0,0 +1,34 @@
+---
+title: AWS Marker report
+---
+# AWS marker report
+
+- Repository: {{ data.meta.repo_url }}
+- Reference Commit: `{{ data.meta.commit_sha }}`
+- Timestamp: `{{ data.meta.timestamp }}`
+
+This is an autogenerated report on our pytest marker usage with a special focus on our AWS compatibility markers, i.e. the ones prefixed with `aws_`.
```

Comment:
The title says that it is only focusing on AWS markers here. Why mentioning others in this sentence, and listing them in the box below?

Reply:
I was pivoting at one point away from making this AWS marker specific, which is why most of the rest of this PR doesn't mention this explicitly. I'll update the title to be more neutral as well :+1: 

It makes more sense to keep this somewhat generic for any kind of marker-based call-to-action.

Reply:
To be honest, I would actually prefer it the other way around. Keep it concise and specific, and if we decide that we want to broaden the scope, we have to add other lists as well (with which we can also adjust the text to be more generic)...